Logger/Student.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Logger;
+public record class Student(FullName StudentName, double GPA, string? Major = null) : IEntity
```

Comment:
```suggestion
public record class Student(FullName StudentName, double GPA, string? Major = null) : BaseEntity
```
All your records should be inheriting from BaseEntity